models/src/Finder.cpp

```diff
@@ -0,0 +1,125 @@
+#include "../include/Finder.hpp"
+#include <RNG.hpp>
+const int NUM_MOVEMENT_STATES = 4;
+const float AGGROTIME = 4;
+const float DEFAULT_SPEED = 2.5;
+const float AUTOSWITCH = 1;
+const bool DEFAULT_WALLPASS = true;
+const int SIGHT_RANGE = 8;
+const sf::Vector2f DEFAULT_START(9.5, 9.5);
+
+Finder::Finder()		
+{
+	_position = (DEFAULT_START);
+	_enemySpeed = (DEFAULT_SPEED);
+	moveState = (EnemyMoveState::north);
+	_switchTime = (AUTOSWITCH);
+	type = (EnemyType::EFinder);
+	_wallPass = (DEFAULT_WALLPASS);
+	changeMoveState();
+}
+Finder::Finder(sf::Vector2f start)		
+{
+	_position = (start);
```

Comment:
Why do these values all have parentheses around them?

Reply:
Parenthesis around most variables are because those variables and values ea are most likely going to change in the future depending where it might be needed. I need more information about what functions are going to be needed so I'm not blindly writing functions for things. So I'm leaving a lot of leeway in my code for those things that need to be adapted

Reply:
I'm not sure how this explains why they have parentheses around them. If it's just a visual indicator for yourself, then it really shouldn't be in code you want to merge to master. Whatever the case, it's not immediately harmful, so I think let's just let it slide.